sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_models_py3.py

```diff
@@ -121,8 +121,8 @@ class BlobHierarchyListSegment(Model):
     }
 
     _attribute_map = {
-        'blob_prefixes': {'key': 'BlobPrefixes', 'type': '[BlobPrefix]', 'xml': {'name': 'BlobPrefixes', 'itemsName': 'BlobPrefix'}},
-        'blob_items': {'key': 'BlobItems', 'type': '[BlobItem]', 'xml': {'name': 'BlobItems', 'itemsName': 'Blob'}},
+        'blob_prefixes': {'key': 'BlobPrefixes', 'type': '[BlobPrefix]', 'xml': {'name': 'BlobPrefix', 'itemsName': 'BlobPrefix'}},
```

Comment:
is this manually added 😀? If so, does that mean we should edit this every time after we generate the code?

Reply:
No, it happens automatically via the transform at https://github.com/Azure/azure-sdk-for-python/blob/46f7acee340c05661ce39616b19ddcf637d0fc20/sdk/storage/azure-storage-blob/swagger/README.md#blobhierarchylistsegment